demisto_sdk/commands/common/git_util.py

```diff
@@ -516,3 +516,15 @@ def get_local_remote_file_path(self, full_file_path: str, tag: str) -> str:
             else:
                 raise exc
         return f'{remote_name}/{tag}:{relative_file_path}'
+
+    def get_all_changed_files(self, prev_ver: str = 'master', committed_only: bool = False,
+                              staged_only: bool = False, debug: bool = False,
+                              include_untracked: bool = False) -> Set[Path]:
+        modified_files = self.modified_files(prev_ver=prev_ver, committed_only=committed_only, staged_only=staged_only,
+                                             debug=debug, include_untracked=include_untracked)
+        added_files = self.added_files(prev_ver=prev_ver, committed_only=committed_only, staged_only=staged_only,
+                                       debug=debug, include_untracked=include_untracked)
+        renamed_files = self.renamed_files(prev_ver=prev_ver, committed_only=committed_only, staged_only=staged_only,
+                                           debug=debug, include_untracked=include_untracked)
+        renamed_files = {file[1] for file in renamed_files}
```

Comment:
perhaps it would be better to add this transformation to `renamed_files()` and control it with a flag ? just thinking this might be needed in another functions as well